packages/ripple-nuxt-tide/README.md

```diff
@@ -2,7 +2,180 @@
 
 > The Nuxt module for adding Tide integration.
 
-## Setup
+## Nuxt configuration
+
+A `nuxt.config.js` file should exist in the root of your nuxt-tide site.
+
+The basic anatomy of a `nuxt.config.js` file is:
+
+```js
+export default {
+  modules: [],
+  debug: false,
+  serverMiddleware: {},
+  head: {},
+  loading: {},
+  router: {},
+  styleResources: {},
+  ripple: {},
+  tide: {}
+}
+```
+
+### modules
+
```

Comment:
Same as this. It's nuxt.js doc not for our module. I don't think we should put it here.

Reply:
Removed.